darrynten/sage-one-php
# Composer style check/fix scripts

## Overview

| Q                 | A
| ----------------- | ---
| Bugfix?           | no
| New feature?      | yes
| Breaking Changes? | no
| Fixed issues      | -

## The problem

We have been relying on StyleCI to do the style checks, and we could save time with a composer script.

## How I resolved it

Took the composer script snippet from the`xero-oauth-php` project.

## How to verify it

`composer update`
`composer check-style`
`composer fix-style`

Notify the following people: @make-it-git
